webui/src/locale/en-US.ts

```
@@ -7,6 +7,8 @@ import loginLocale from '@/views/login/locale/en-US'
 import oobeLocale from '@/views/oobe/locale/en-US'
 import topBanPageLocale from '@/views/ranks/locale/en-US'
 import ruleManageMentLocale from '@/views/rule-management/locale/en-US'
+import chartsLocale from '@/views/charts/locale/en-US'
```

Biome

[error] Shouldn't redeclare 'chartsLocale'. Consider to delete it or rename it.

'chartsLocale' is defined here:


(lint/suspicious/noRedeclare)